tests/integration/helm_otc_metadata_installation_test.go

```diff
@@ -146,6 +147,16 @@ func Test_Helm_OT_Metadata(t *testing.T) {
 				require.EqualValues(t, 0, daemonsets[0].Status.NumberUnavailable)
 				return ctx
 			}).
+		Assess("metrics are present",
```

Comment:
Let's add a TODO to extract this out to a separate feature when we figure out sharing a cluster between features.